src/main/java/com/faforever/client/replay/ReplayService.java

```diff
@@ -186,8 +207,8 @@ protected Thread startDirectoryWatcher(Path replaysDirectory) throws IOException
 
   @VisibleForTesting
   protected void onLocalReplaysWatchEvent(WatchKey key) {
```

Comment:
Have u thought about what happens if a user drops a new replay into his replay folder... that is what happens here.

Reply:
Maybe u can simplify this method by just going back to the first page :thinking: